src/swarm-tui/Swarm/TUI/Controller.hs

```diff
@@ -1,6 +1,9 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE QuasiQuotes #-}
 {-# LANGUAGE RecordWildCards #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+
+{-# HLINT ignore "Use fewer imports" #-}
```

Comment:
What is this HLINT pragma for?

Reply:
That's what HLS does if you by mistake click "ignore".